interview-questions.md

```diff
@@ -8,50 +8,52 @@ Then, research the question to expand on your answer. Even if you feel you have
 
 INSTRUCTOR EXAMPLE: What is a conditional statement?
 
-Your answer:
+Your answer: 
 
 Researched answer:
 
 1. What is git? What is the difference between git and Github?
 
-Your answer:
+Your answer: Git stands for Global Information Tracker and gives you a way to access information remotely or upload remotely to a "hub" of some sort.  Github is a place where you can use git to send files documents and data to a central hub to be accesses.
 
-Researched answer:
+Git is the tool that lets you move things around such as files and documents and Github is the place to hold said files or to pull files and data from.
+
+Researched answer: Git is an open source code management that helps companies and people track changes in code and actually helps multiple web developers to work on projects together. Git is a way to track changes in your code while Github is a cloud-based service that lets you manage and use git repositories.
 
 2. Which JavaScript operators will return a Boolean value?
 
-Your answer:
+Your answer: The javascript operators that will return a Boolean value are AND (&&), OR (||), and NOT (!). 
 
-Researched answer:
+Researched answer: After reasearching I keep coming up with the same answer that the JavaScript operators that will return a Boolean value are, AND (&&), OR(||), and NOT (!).
```

Comment:
This was looking at the categories of operators that produce a boolean value. You have examples of logical operators. Look at the other two in the syllabus: equality and relational
https://github.com/learn-academy-2022-echo/Syllabus/blob/main/javascript/intro.md
https://github.com/learn-academy-2022-echo/Syllabus/blob/main/javascript/conditionals.md